cli/tests/e2e/snapshots/test_ci/test_nosem/autofix---disable-nosem/output.txt

```diff
@@ -10,7 +10,7 @@ SEMGREP_APP_TOKEN="" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<
 
 
 ┌───────────────────────────┐
-│ 12 Blocking Code Findings │
+│ 10 Blocking Code Findings │
```

Comment:
Why the change in this snapshot? Seems unrelated to the PR.

Reply:
This is a side effect of the tests now consistently using the same ignored finding match based ID's - I think it's worth letting this change just since it makes the tests a bit more consistent.

I confirmed that the disabled vs enabled cases still have the coverage we want.